docs/source/depth_estimation.mdx

```diff
@@ -121,7 +121,7 @@ You can also visualize several different images and their corresponding depth ma
 ```
 
 <div class="flex justify-center">
-    <img src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/datasets/depth_est_collage.png">
+    <img src="https://i.postimg.cc/fLWRs5SN/depth-est-collage.png">
```

Comment:
Open a PR adding these assets as Git-LFS here: https://huggingface.co/datasets/huggingface/documentation-images/

Don't forget to optimize the images so that the page loading is optimal (you can use a tool like ImageOptim for this).

Reply:
Will do

Reply:
Regarding this, I can't seem to clone the repo in the local or remote system, getting this error,
```
Cloning into 'documentation-images'...
error: RPC failed; HTTP 504 curl 22 The requested URL returned error: 504
fatal: the remote end hung up unexpectedly
```

Reply:
Maybe because of the platform outage. Could you try now?

Reply:
Yes it worked now

Reply:
Created the PR, https://huggingface.co/datasets/huggingface/documentation-images/discussions/40

Reply:
@sayakpaul  I just realized I forgot to use lfs in my PR but it has already been merged. Should I create another PR?

Reply:
@sayakpaul should I create another PR on https://huggingface.co/datasets/huggingface/documentation-images with lfs?

Reply:
Sure.

Reply:
@sayakpaul  created the PR, https://huggingface.co/datasets/huggingface/documentation-images/discussions/41